src/components/widgets/WebWidget.tsx

```diff
@@ -0,0 +1,28 @@
+import { useNavigation } from '@react-navigation/core';
+import React, { useCallback } from 'react';
+
+import { Icon, normalize } from '../../config';
+import { ScreenName, WidgetProps } from '../../types';
+import { Image } from '../Image';
+
+import { DefaultWidget } from './DefaultWidget';
+
+export const WebWidget = ({ text = '', additionalProps }: WidgetProps) => {
+  const navigation = useNavigation();
+
+  const onPress = useCallback(() => {
+    navigation.navigate(ScreenName.Web, {
+      title: additionalProps?.staticContentTitle,
+      webUrl: additionalProps?.webUrl
+    });
+  }, [navigation, text]);
+
+  return (
+    <DefaultWidget
+      Icon={() => <Icon.Url size={normalize(26)} />}
```

Comment:
looking at survey widget, there is a difference: `Icon={(props) => <Icon.Surveys {...props} size={normalize(22)} />}` so i guess we need to pass the props through here as well?

Reply:
Yes, that's right. I'll add it right away.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/30ddb5e86e2a4a1b2abc6ba74ca80a359c6df851